.gitignore

```diff
@@ -25,3 +25,4 @@ cache/
 tmp-*.json
 draft*signer*.json
 ready-*
+broadcast/NestedSignFromJson.s.sol/1
```

Comment:
just curious, why is this needed?

Reply:
Just to avoid people to push all the broadcast tx that are created and will be accidentally pushed.